backend/spec/spec_helper.rb

```diff
@@ -47,7 +47,7 @@
 Capybara.javascript_driver = :poltergeist
 
 # Set timeout to something high enough to allow CI to pass
-Capybara.default_wait_time = 10
+Capybara.default_wait_time = 35
```

Comment:
you running build.sh ?  my air has no issues with it that way

On Mon, Sep 14, 2015 at 6:04 PM, Jason Fleetwood-Boldt <
notifications@github.com> wrote:

> In backend/spec/spec_helper.rb
> <https://github.com/spree/spree/pull/6731#discussion_r39453394>:
>
> > @@ -47,7 +47,7 @@
> >  Capybara.javascript_driver = :poltergeist
> >
> >  # Set timeout to something high enough to allow CI to pass
> > -Capybara.default_wait_time = 10
> > +Capybara.default_wait_time = 35
>
> Sorry I swear my MacBook Air can't finish the suite without raising these.
> I had meant to *not* check these in, I will revert
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spree/spree/pull/6731/files#r39453394>.
>